tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -30,15 +30,16 @@
 #define TF2_EIGEN_H
```

Comment:
As a meta-comment in this file, I changed this to use Deanna's toMsg()/fromMsg() timestamp conversion, which cleans up the file nicely.  I also removed any changes dealing with whitespace or style; this is to take care of Tully's concern w.r.t. cherry-picking from the ROS1 geometry2.